packages/inventory/src/InventoryDetail.js

```diff
@@ -24,10 +24,10 @@ class InventoryDetail extends React.Component {
     }
 
     render() {
-        const { root, match: { params }, useCard, hideBack, actions, showTags } = this.props;
+        const { root, match: { params }, useCard, hideBack, actions, showTags, hideInvLink } = this.props;
         return (
             <React.Fragment>
-                <Entitydetail useCard={ useCard } actions={ actions } showTags={ showTags } />
+                <Entitydetail useCard={ useCard } actions={ actions } showTags={ showTags } hideInvLink={ hideInvLink } />
```

Comment:
What about to do `{...rest}` ?

Reply:
We don't want send rest props, this is just hot fix to allow hiding inventory app link.